cmds/tfuser/cmds_network.go

```diff
@@ -96,12 +96,12 @@ func cmdsAddNode(c *cli.Context) error {
 		return errors.Wrapf(err, "failed to add a node to the network %s", network.Name)
 	}
 
-	f, err = os.Open(networkSchema)
```

Comment:
Looks good, we should also do that for the other commands as well

Reply:
There was only one another from what I saw, it's done